Exercise_C/index.css

```diff
@@ -2,3 +2,29 @@
 This css sheet is linked to index.html
 You can style tags, classes (.) or ids (#) from this file.
 */
+div {
+    height: 100px;
+    width: 100px;
+}
+
+#blue {
+    background-color: blue;
+}
+
+#green {
+    background-color: green;
+    
+}
+
+#red {
+    background-color: red;
+}
+
+section {
+    display: flex;
+    justify-content: space-between;
+}
+
+.center {
+    justify-content: center;
+}
```

Comment:
That's an excellent use of flex. This shows that you have a decent understanding of how flex works